main/config.py

```diff
@@ -21,6 +26,14 @@ class Config:
     FIREBASE_EMULATOR_TEST_HOST = typing.cast("str | None", settings.FIREBASE_EMULATOR_TEST_HOST)
     FIREBASE_EMULATOR_HOST = os.environ.get("FIREBASE_DATABASE_EMULATOR_HOST")
 
+    ENABLE_DANGER_MODE = typing.cast("bool", settings.ENABLE_DANGER_MODE)
```

Comment:
@Rup-Narayan-Rajbanshi When using a environment variable, let's define it here and then reference it from there.

Directly accessing these values from settings is not typesafe.